### PR TITLE
Create kosheri.tex

### PR DESCRIPTION
**Pull request by MaxwellBo**

### CODE REVIEW

File: recipes/kosheri.tex [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Place the lentils in a large sieve and wash them under a cold running tap. Transfer to a large saucepan, cover with plenty of cold water, and bring to a boil. Lower the heat and simmer for 25 minutes. The lentils should be tender but far from mushy. Drain in a colander and set aside.
+
+\Ing{200g basmati rice}
+\Ing{40g unsalted butter}
+\Ing{50g vermicelli noodles, broken into 4cm pieces}
+\Ing{400ml chicken stock or water}
+\Ing{\frac{1}{2} tsp grated nutmeg}
+\Ing{1 & \frac{1}{2}tsp ground cinnamon}
+\Ing{1 & \frac{1}{2}tsp salt}
+\Ing{\frac{1}{2}tsp black pepper}
```
Comment (TRManderson) [inline recipes/kosheri.tex]: All these are math mode fractions, use `\fr` in recipes

File: recipes/kosheri.tex [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+To make the sauce, heat the olive oil in a saucepan, add the garlic and chills and fry for 2 minutes. Add the chopped tomatoes, water, vinegar, salt, and cumin. Bring to a boil, then lower the heat and simmer for 20 minutes, until slightly thickened. Remove the sauce from the heat, stir in the cilantro, and then taste. See if you want to add any salt, pepper, or extra cilantro. Keep hot or leave to cool; both ways will work with the hot kosheri. Just remember to adjust the seasoning again when cold.
+\Ing{300g green lentils}
+Place the lentils in a large sieve and wash them under a cold running tap. Transfer to a large saucepan, cover with plenty of cold water, and bring to a boil. Lower the heat and simmer for 25 minutes. The lentils should be tender but far from mushy. Drain in a colander and set aside.
+
+\Ing{200g basmati rice}
+\Ing{40g unsalted butter}
+\Ing{50g vermicelli noodles, broken into 4cm pieces}
+\Ing{400ml chicken stock or water}
+\Ing{\frac{1}{2} tsp grated nutmeg}
+\Ing{1 & \frac{1}{2}tsp ground cinnamon}
```
Comment (TRManderson) [inline recipes/kosheri.tex]: Escape ampersand

File: recipes/kosheri.tex [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+\Ing{3 tsp salt}
+\Ing{2 tsp ground cumin}
+\Ing{20g coriander leaves, chopped}
+To make the sauce, heat the olive oil in a saucepan, add the garlic and chills and fry for 2 minutes. Add the chopped tomatoes, water, vinegar, salt, and cumin. Bring to a boil, then lower the heat and simmer for 20 minutes, until slightly thickened. Remove the sauce from the heat, stir in the cilantro, and then taste. See if you want to add any salt, pepper, or extra cilantro. Keep hot or leave to cool; both ways will work with the hot kosheri. Just remember to adjust the seasoning again when cold.
+\Ing{300g green lentils}
+Place the lentils in a large sieve and wash them under a cold running tap. Transfer to a large saucepan, cover with plenty of cold water, and bring to a boil. Lower the heat and simmer for 25 minutes. The lentils should be tender but far from mushy. Drain in a colander and set aside.
+
+\Ing{200g basmati rice}
+\Ing{40g unsalted butter}
+\Ing{50g vermicelli noodles, broken into 4cm pieces}
```
Comment (TRManderson) [inline recipes/kosheri.tex]: Really only 50g? 50g isn't that much

Comment (MaxwellBo) [inline recipes/kosheri.tex]: The noodles are quite light, that's quite a lot of noodles

**Pull request merged**